### ISSUE TITLE
how to train from the begin

### ISSUE THREAD

**Issue by ChChwang** [MhLiao/MaskTextSpotter#33]
The pretrain code loaded the trained model to get the model, how to train from the begin without downloaded model? 
the loss seems to be very small.

**Comment by MhLiao** [comment]
You can pre-train the model with SynthText.

**Comment by Shualite** [comment]
SynthText only has '.mat' for ground truth labeling. How can I know the structure of file '.txt' for gt labeling?

**Comment by Shualite** [comment]
@MhLiao

**Comment by MhLiao** [comment]
@Shualite I have provided the converted labels of SynthText [here](https://github.com/MhLiao/MaskTextSpotter#datasets).

**Comment by ChChwang** [comment]
@MhLiao the cfg.SOLVER.RESUME shoule be set False when pretraining?
And the model you provided was evaluated, and the F1 is 0.80?
thanks a lot

**Comment by ChChwang** [comment]
@MhLiao the cfg.SOLVER.RESUME shoule be set False when pretraining?
And the model you provided was evaluated, and the F1 is 0.80, because of it's epoch is 150k?
thanks a lot

**Comment by sno6** [comment]
Has anybody had any luck with training a custom model completely from scratch? can I simply provide a model such as the finetuned weights and then set RESUME to false or is there more to it?
